test/dotnet-migrate.Tests/GivenThatIWantToMigrateDeprecatedProjects.cs

```diff
@@ -406,7 +406,7 @@ public void WhenMigratingDeprecatedResourceOptionsWarningsArePrinted()
                 "The 'resourceFiles' option is deprecated. Use 'embed' in 'buildOptions' instead.");
         }
 
-        [Fact]
+        [RequiresSpecificFrameworkFact("netcoreapp1.0")]
```

Comment:
Are you sure you want this?  We can run part of these tests (at least migrate/restore/build) without the shared framework.